it.tests/src/main/java/com/venia/it/tests/CategoryPageIT.java

```diff
@@ -36,7 +36,8 @@
 
     @Test
     public void testProductListPageWithSampleData() throws ClientException, IOException {
-        String pagePath = VENIA_CONTENT_US_EN_PRODUCTS_CATEGORY_PAGE + ".uid-1.html";
+
+        String pagePath = VENIA_CONTENT_US_EN_PRODUCTS_CATEGORY_PAGE + ".html/outdoor";
```

Comment:
minor but the url does not fit the urlprovider configuration (applies to all)

the ITs still work as this is only about resolution I guess (?)

Reply:
Fixed the urls to stay in sync with the configured url format.